src/zope/sendmail/tests/test_queue.py

```diff
@@ -139,7 +139,12 @@ def test_makeMaildir_creates(self):
         self.assertTrue(os.path.exists(md.path))
 
     def test_threadName(self):
-        self.assertEqual(self.thread.getName(),
+        try:
+            thread_name = self.thread.name
```

Comment:
Thread.name is new in Python 2.6, see https://docs.python.org/2.7/library/threading.html#threading.Thread.name

So there I see no need for the try-except.

Reply:
Oh indeed! The two methods have been deprecated recently but they basically were redundant since ages.
I will take care of that.